sshuttle/client.py

```diff
@@ -414,6 +419,11 @@ def expire_connections(now, mux):
                 pass
 
     tcp_conns = new_tcp_conns
+    global _allowed_targets_modified
```

Comment:
confusing... you have the exact same variable names in reload_acl_files function  are those global or local?  if global... define only once, and move to the top of the file.

Reply:
global here is to say that you are using the global variable and not creating a new local variable.   This is the way python does it.